lib/src/ui/watch/tv_screen.dart

```diff
@@ -48,10 +110,26 @@ class _TvScreenState extends ConsumerState<TvScreen>
 
   Widget _iosBuilder(
     BuildContext context,
+    WidgetRef ref,
   ) {
+    final ChannelVariant selectedVariant = ref.watch(variantStateProvider);
     return CupertinoPageScaffold(
       navigationBar: CupertinoNavigationBar(
-        middle: const Text('Top Rated'),
+        middle: InkWell(
```

Comment:
should use `AppBarTextButton` from widget/buttons.dart, especially since InkWell is and material widget and we want to use Cupertino ones on iOS.